docs/user/openstack/external_load_balancer.md

```diff
@@ -0,0 +1,224 @@
+# Deploying OpenShift with an external load balancer
```

Comment:
Should we adopt the `User managed LB` and `OpenShift managed LB` terminology here too, instead of external/internal LB?